tests/cypress/e2e/unit_tests/first_login_rancher.spec.ts

```diff
@@ -13,14 +12,11 @@ limitations under the License.
 */
 
 import * as cypressLib from '@rancher-ecp-qa/cypress-library';
-import { qase } from 'cypress-qase-reporter/dist/mocha';
 
 
 Cypress.config();
 describe('First login on Rancher', () => {
-  qase(46,
     it('Log in and accept terms and conditions', () => {
     cypressLib.firstLogin();
     })
-  );
-})
+})
```

Comment:
I need to add new line at the end of this file, I will add it after QASE integration PR gets in.